winml/lib/Api.Ort/OnnxruntimeEnvironment.cpp

```diff
@@ -7,10 +7,65 @@
 #include "core/platform/windows/TraceLoggingConfig.h"
 #include <evntrace.h>
 
+#include <windows.h>
+
 using namespace _winml;
 
 static bool debug_output_ = false;
 
+static HRESULT GetOnnxruntimeLibrary(HMODULE& module) {
+  std::string system_path;
+#ifdef BUILD_INBOX
+  system_path.reserve(MAX_PATH);
+  if (0 >= GetSystemDirectoryA(system_path.data(), system_path.size())) {
+    return HRESULT_FROM_WIN32(GetLastError());
+  }
+  system_path += "\\onnxruntime.dll";
+#else
+  system_path = "onnxruntime.dll";
+#endif
+
+  auto out_module = LoadLibraryA(system_path.data());
```

Comment:
* Shouldn't you FreeLibrary this somewhere?
* Should we save it somewhere instead of loading it in GetVersionedOrtApi and again in GetVersionedWinmlAdapterApi?
* This is desktop only, how about UWP/Store?

Reply:
updated to loadlibraryexa with the system32 option flag like you suggested.